### PR TITLE
added PHP code generator

### PR DESCRIPTION
**Pull request by StrikerRUS**
PHP is one of the most popular programming language for web development. So, use cases are quite obvious - online model serving.

### CODE REVIEW

**Comment by izeigerman** [comment]
Fantastic, thank you! Will take a look 👍

File: m2cgen/interpreters/php/interpreter.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            self._cg.add_return_statement(last_result)
+
+        if self.with_linear_algebra:
+            filename = os.path.join(
+                os.path.dirname(__file__), "linear_algebra.php")
+            self._cg.prepend_code_lines(utils.get_file_content(filename))
+
+        self._cg.prepend_code_line("<?php")
+
+        return self._cg.code
```
Comment (izeigerman) [inline m2cgen/interpreters/php/interpreter.py]: Not a PHP expert but shall the code block be closed with `?>`?

File: m2cgen/interpreters/powershell/code_generator.py [hunk 2]
```diff
@@ -45,9 +45,7 @@ def math_function_invocation(self, function_name, *args):
         return function_name + "(" + ", ".join(map(str, args)) + ")"
 
     def get_var_name(self):
-        var_name = "$var" + str(self._var_idx)
-        self._var_idx += 1
-        return var_name
+        return "$" + super().get_var_name()
```
Comment (izeigerman) [inline m2cgen/interpreters/powershell/code_generator.py]: Thanks 👍

File: m2cgen/interpreters/php/interpreter.py (hunk 1 above)
Comment (StrikerRUS) [reply m2cgen/interpreters/php/interpreter.py]: > code block

Yes.

Entire ".php" file - no.

Refer to https://stackoverflow.com/a/19953547 and https://stackoverflow.com/a/4499749

> Finally, many PHP frameworks including Symfony, Zend and Laravel (there is no mention of this in the coding guidelines but it follows the suit) and the PSR-2 standard (item 2.2) require omission of the closing tag. PHP manual itself (1,2), Wordpress, Drupal and many other PHP software I guess, advise to do so.

Comment (izeigerman) [reply m2cgen/interpreters/php/interpreter.py]: Thanks!

**Pull request merged**